examples/analog_hamiltonian_simulation/04_Running_Analog_Hamiltonian_Simulation_with_local_simulator.ipynb

```diff
@@ -0,0 +1,482 @@
+{
```

Comment:
<p>This is the first time we mention Hilbert spaces. Previously, we mentioned only Hamiltonians. I'm not against speaking about Hilbert spaces, I suggest either of the following two options:</p><ol><li>Introduce the idea of truncated Hilbert space in the same paragraph where we speak about full and approximate Hamiltonians</li><li>Or, stick to explaining everything in terms of Hamiltonians (full vs approximated)</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>